deploy-as-code/helm/environments/ifix-qa.yaml

```diff
@@ -77,6 +77,7 @@ fiscal-event-post-processor:
 ifix-es-pipeline:
   fiscal-event-kafka-push-topic: "fiscal-event-request-validated"
   fiscal-event-es-push-topic: "index-fiscal-events"
+  ifix-department-entity-search-endpoint: "ifix-department-entity/departmentEntity/hierarchyLevel/v1/_search"
```

Comment:
This will give you the `/hierarchyLevel` metadata, not the actual department entities. Have you tested this locally?

Reply:
yes, I have port-forwarded and ran the flow locally. It is fetching the hierarchy level and their label data which I am using for enrichment